x-pack/test/security_solution_endpoint/page_objects/hosts_page.ts

```diff
@@ -105,5 +108,55 @@ export function SecurityHostsPageProvider({ getService, getPageObjects }: FtrPro
       await deleteMetadataStream(getService);
       await deleteTelemetryStream(getService);
     },
+    /**
+     * Runs Nodes Events
+     */
+    async runNodeEvents() {
+      await testSubjects.exists('resolver:submenu:button', { timeout: 400 });
+      const NodeSubmenuButtons = await testSubjects.findAll('resolver:submenu:button');
+      for (let b = 0; b < NodeSubmenuButtons.length; b++) {
+        await (await testSubjects.findAll('resolver:submenu:button'))[b].click();
+      }
+      await testSubjects.exists('resolver:map:node-submenu-item', { timeout: 400 });
+      const NodeSubmenuItems = await testSubjects.findAll('resolver:map:node-submenu-item');
+      for (let i = 0; i < NodeSubmenuItems.length; i++) {
+        await (await testSubjects.findAll('resolver:map:node-submenu-item'))[i].click();
+        const Events = await testSubjects.findAll('resolver:map:node-submenu-item');
+        const EventText = await Events[i]._webElement.getText();
+        const LinkText = await testSubjects.find('resolver:breadcrumbs:last');
+        const linkText = await LinkText._webElement.getText();
+        // console.log(linkText);
+        expect(EventText).to.equal(linkText);
+      }
+    },
+    /**
+     * Navigate to Events Panel
+     */
+    async navigateToEventsPanel() {
+      if (!(await testSubjects.exists('investigate-in-resolver-button', { timeout: 400 }))) {
+        await (await testSubjects.find('navigation-hosts')).click();
+        await testSubjects.click('navigation-events');
+        await testSubjects.existOrFail('event');
+      }
+    },
+    /**
+     * execute Query And Open Resolver
+     */
+    async executeQueryAndOpenResolver(query: string) {
+      await queryBar.setQuery(query);
+      await queryBar.submitQuery();
+      await testSubjects.click('full-screen');
+      await testSubjects.click('investigate-in-resolver-button');
+      await testSubjects.click('full-screen');
+    },
+    /**
+     * Dismiss Banner if available.
+     */
+    async dismissBanner() {
```

Comment:
I think this method should go in a more generic location since it's used across all of Kibana.  In fact, I need to use it in a test as well.

Maybe in `test/functional/page_objects/common_page.ts`

Your test is already using PageObjects.common so it wouldn't require any additional page objects to be pulled in.

Reply:
Ok I will put in there

Reply:
Done